extensions/amp-story/1.0/amp-story.css

```diff
@@ -655,6 +671,38 @@ amp-story .amp-video-eq,
   z-index: 10000 !important;
 }
 
+.i-amphtml-story-expanded-view-overflow {
+  top: 0 !important;
+  left: 0 !important;
+  width: 100% !important;
+  height: 100% !important;
+  background-color: rgba(0, 0, 0, 1) !important;
+  z-index: 1 !important;
+  position: absolute;
+}
+
+.i-amphtml-expanded-view-close-button {
+  position: absolute !important;
+  top: 0 !important;
+  left: 0 !important;
+  height: 48px !important;
+  width: 48px !important;
+  color: #757575 !important;
```

Comment:
The contrast is too low (4.56). I haven't seen the mocks in a while, but this should be brighter to respect the contrast ratio required by the material guidelines.

Reply:
Done.